world_boss/app/kms.py

```diff
@@ -39,11 +40,13 @@ def address(self) -> str:
         return ethereum_kms_signer.get_eth_address(self._key_id)
 
     def _get_client(self, headless_url: str) -> Client:
-        transport = RequestsHTTPTransport(url=headless_url)
+        transport = RequestsHTTPTransport(
+            url=headless_url, headers=get_jwt_auth_header()
```

Comment:
Considering that you will need another header in the future, how about creating header object at the other place and add "Authorization" header into it?

Reply:
we don't have any other headers to use right now, so I'll fix them later when needed.